.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Feature Request
+about: Suggest an idea for the roadmap! The team will review
+title: "[request]: describe request here"
```

Comment:
thoughts on "Feature Request" vs "Request"? I'd also make this `Capital Case`

Reply:
Request is probably better for the near term as I'd prefer to make it easier for more folks to use the template than not.  I'll update